soc/arm/microchip_mec/mec1501/power.c

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2019 Microchip Technology Inc.
+ * Copyright (c) 2016 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <sys_io.h>
+#include <misc/__assert.h>
+#include <power.h>
+#include <soc.h>
+#include "device_power.h"
+
+#if defined(CONFIG_SYS_POWER_DEEP_SLEEP_STATES)
+
+/*
+ * Deep Sleep
+ * Pros:
+ * Lower power dissipation, 48MHz PLL is off
+ * Cons:
```

Comment:
so for deep sleep and light sleep, the difference is that the PLL(48Mhz) for CPU clock is on or off? If yes, can I understand that CPU just have different clock for deep/light sleep? @scottwcpg